File: Chapter05/code/Exercise05_10.py
from pyspark.sql import Row
rowRDD = map(lambda x: Row(*x), data)

from pyspark.sql.types import StructType, StructField, StringType
schema = StructType([StructField("city", StringType(), True), StructField("zip_code", StringType(), True)])

dfToRdd04 = spark.createDataFrame(rowRDD, schema)
dfToRdd04.printSchema()

File: Chapter05/code/Exercise05_10.scala
import org.apache.spark.sql.Row
val rowRDD = rdd.map(record => Row(record._1, record._2))

val schema = StructType(columns.map(fieldName => StructField(fieldName, StringType, nullable = true)))

val dfToRdd04 = spark.createDataFrame(rowRDD, schema)
dfToRdd04.printSchema()
